docs/INSTALL-devel-flowchart.txt

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
We had decided that the change from haas to HIL will be made in a dedicated PR. There were some good reasons for that.
It gets confusing when everything is referred to as HIL and then we have to configure
`haas.cfg` and to spawn a server it is `haas serve <port no>` and `haas serve networks`
and all other commands start with `haas`.

IMO, we instructions should change the last. Other functional components should be changed first.